src/SocialAuthUserManager.php

```diff
@@ -170,14 +168,14 @@ public function setSessionKeysToNullify(array $session_keys) {
    * @return \Symfony\Component\HttpFoundation\RedirectResponse
    *   A redirect response.
    */
-  public function authenticateUser($name, $email, $pluginId, $provider_user_id, $token, $picture_url = FALSE, $data = '') {
+  public function authenticateUser($name, $email, $provider_user_id, $token, $picture_url = FALSE, $data = '') {
 
     // Checks for record in social _auth entity.
-    $user_exist = $this->checkIfUserExists($pluginId, $provider_user_id);
+    $user_exist = $this->checkIfUserExists($this->pluginId, $provider_user_id);
```

Comment:
Why instead of passing $pluginId, you just access $this->pluginId in method function? It will make it even easier in your implementer controller to check if user exists before storing additional data.
You should remove $pluginId parameter for all the methods where this update can be applied